src/addTaskUI.js

```diff
@@ -16,21 +16,17 @@ const appendTask = (item, list) => {
       moveImg.src = trashSrc;
```

Comment:
- [ ] Please, consider implementing the `filter()` method for the "Clear all completed" button, as it is mentioned in the project requirements. As I understand, you are using `splice` from "clearTask" function 
<img width="398" alt="Screen Shot 2022-06-03 at 10 01 08" src="https://user-images.githubusercontent.com/49064106/171795924-1e7c3cc2-006d-4167-8c76-551b0278ec81.png">
 
> clearTask function
<img width="278" alt="Screen Shot 2022-06-03 at 10 04 07" src="https://user-images.githubusercontent.com/49064106/171796303-90302857-c9ce-4c08-aafd-047d4ab89f6b.png">